### DISCUSSION TITLE
Any possibility for an exe version? (not ms store)

### DISCUSSION THREAD

**Discussion by Nomad-0** [File-New-Project/EarTrumpet#893] [Q&A] [answered]
I prefer exe installer to ms store apps due to some difficulties with ms store. Is there any possibility that an installer version would be released?

**Comment by riverar** [comment, marked as answer]
@Nomad-0 Hey there, we also have EarTrumpet on Chocolatey and winget. https://eartrumpet.app

**Reply by Nomad-0** [reply]
Thanks, but any chance you guys will consider an independent installer in the near future?

**Reply by riverar** [reply]
Sorry, no. Because then we'll have to build update infrastructure to keep folks up to date. Then there's figuring out where to put the binaries, how to keep it updated, etc. etc.

**Reply by Nomad-0** [reply]
oh ok, thank you
